projects/1.19/assets/beachparty/beachparty/lang/zh_cn.json

```
@@ -0,0 +1,131 @@
+{
+  "beachparty.container.cabinet": "橱柜",
+  "block.beachparty.beach_chair":"沙滩摇椅",
+  "block.beachparty.beach_grass": "沙草",
+  "block.beachparty.beach_towel": "沙滩毛巾",
+  "block.beachparty.cabinet": "棕榈橱柜",
+  "block.beachparty.chair":"棕榈布艺椅子",
+  "block.beachparty.cocoa_cocktail": "§a绿野仙踪",

```

```suggestion
  "block.beachparty.cocoa_cocktail": "可可鸡尾酒",
```
绿野仙踪的做法里貌似也没有可可吧
是游戏内有给出做法吗

---

这部分自用版本，我忘了修改，其实应该更书面语的，自用版本是为了更贴切其增益效果及浪漫而产生的意译，确实应当修改。